src/box/lua/net_box.lua

```diff
@@ -167,11 +168,12 @@ local function create_transport(host, port, user, password, callback,
     -- response anymore.
     -- Sync requests are implemented as async call + immediate
     -- wait for a result.
-    local requests         = internal.new_registry()
+    local requests          = internal.new_registry()
 
     local worker_fiber
-    local send_buf         = buffer.ibuf(buffer.READAHEAD)
-    local recv_buf         = buffer.ibuf(buffer.READAHEAD)
+    local send_buf          = buffer.ibuf(buffer.READAHEAD)
+    local recv_buf          = buffer.ibuf(buffer.READAHEAD)
+    local is_closing        = false
```

Comment:
Please add a brief comment to this variable.

Reply:
fixed